<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeProject</class>
 <widget class="QMainWindow" name="MakeProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:        </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name_input"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Path:          </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="project_path_input">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="project_path_search_button">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Video path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="video_path_input">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="video_path_search_button">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="ok_button">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_button">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
